bot/cogs/clean.py

```diff
@@ -114,43 +113,37 @@ def predicate_regex(message: Message) -> bool:
         if not channels:
             channels = [ctx.channel]
 
+        # Delete the invocation first
+        with self.mod_log.ignore_all():
+            await ctx.message.delete()
```

Comment:
Let's still try to use `ignore()` when feasible. We don't need to search to get the ID for this message, so it's not a problem.

Reply:
Alright, fair enough.